source/Ship.cpp

```diff
@@ -2453,7 +2453,7 @@ void Ship::Launch(list<shared_ptr<Ship>> &ships, vector<Visual> &visuals)
 			bay.ship->SetParent(shared_from_this());
 			bay.ship->UnmarkForRemoval();
 			// Update the cached sum of carried ship masses.
-			carriedMass -= bay.ship->Mass();
+			carriedMass -= bay.ship->InertialMass();
```

Comment:
I'm not too sure on using the InertialMass of carried ships. I think this breaks down with ammo transfer. Looking farther up from here:
```c++
auto takenAmmo = TransferAmmo(toRestock, *this, *bay.ship);
bool tookAmmo = !takenAmmo.empty();
if(tookAmmo)
{
	// Update the carried mass cache.
	for(auto &&item : takenAmmo)
		carriedMass += item.first->Mass() * item.second;
}
```
Ammo is taken from the mother ship's installed outfit and added to the carried ship's installed outfits, but the manner in which this is reflected in carriedMass is that the mass of the ammo is removed from the mother ship's installed and added directly to the carriedMass. The reason this breaks is because this ammo is being installed on the carried ship, meaning the carried ship's inertial mass should have an effect. So either we shouldn't use the inertial mass of carried ships, or ammo transfer should take the carried ship's inertial reduction into account.

Reply:
I would suggest carried ships simply use their parent's inertia reduction stat whenever doing their... whatevers. I'd have to look more closely to confirm that's reasonable.